pom.xml

```diff
@@ -66,13 +67,35 @@
 			<artifactId>json</artifactId>
 			<version>20211205</version>
 		</dependency>
+
+    <!-- optional GSON, used for JSON-commands -->
 		<dependency>
 			<groupId>com.google.code.gson</groupId>
 			<artifactId>gson</artifactId>
-			<version>2.8.9</version>
+			<version>2.9.0</version>
+      <scope>provided</scope>
 		</dependency>
+    <!-- optional Jackson, used for JSON-commands -->
+    <dependency>
+      <groupId>com.fasterxml.jackson.core</groupId>
+      <artifactId>jackson-databind</artifactId>
+      <version>${jackson.version}</version>
```

Comment:
suggest 2.13.2.2

![image](https://user-images.githubusercontent.com/17929800/161434536-d9473dc8-9242-4a0d-8e45-a56bff3954bf.png)

Reply:
@sebdehne you need to add a separate `jackson.databind.version` param, since for now, there's a specific patch version of jackson-databind for a vulnerability fix.

Reply:
Thanks for the heads-up, but as soon as I change it to the latest, a new version will be release. The whole point of this change is that Jedis shall NOT require/dictate any version. Therefore the dependency is marked as <scope>provided</scope>. So there is no need really to keep up with the latest version.